airtable/plugin.py

```diff
@@ -146,28 +157,65 @@ class AirtableRecord:
 
 
 def airtable_record_list(
-    base_id: AirtableBaseID, table_id: AirtableTableID
+    base_id: AirtableBaseID,
+    table_id: AirtableTableID,
+    pick: Optional[set[str]],
 ) -> list[AirtableRecord]:
     """
-    Return results of an Airtable `list records` API call.
+    Return results of an Airtable `list records` API call with the option of picking
+    fields from the records.  Picked fields are required, so use it to ensure that the
+    records have the fields you need.
```

Comment:
```suggestion
     Return results of an Airtable `list records` API call with the option of
     ensuring that required fields exists. If the required fields don't exist 
     in the schema, an error is thrown.
```

Reply:
@vrv can i get you to take a pass at this?

Reply:
fields_to_include is what's suggested by chatgpt .

```
Another alternative could be required_fields, emphasizing that these fields are necessary for the caller's use case. However, fields_to_include seems to balance both clarity and functionality better, as it does not strictly suggest that the fields are required for the function to operate but rather are required by the caller for their specific purpose.
```